src/content/release/breaking-changes/image-filter-blur-tilemode.md

```diff
@@ -0,0 +1,83 @@
+---
+title: ImageFilter.blur default tile mode automatic selection.
+description: >-
+  If a tile mode wasn't specified in the constructor, ImageFilter.blur will
+  select one based on the rendering context.
+---
+
+## Summary
+
+  The `ui.ImageFilter.blur`'s default tile mode is now automatically selected
+  by the backend. Previously `TileMode.clamp` was used unless a different tile
+  mode was specified. Now, the default is `null` and specifies automatic
+  selection unless a specific tile mode is specified.
+
+## Background
+
+  `ImageFilter.blur`'s _tile mode_ specifies what happens to edge pixels for
+  the applied filter. There are four options, `TileMode.clamp` (the previous
+  default), `Tilemode.repeated`, `TileMode.mirror`, and `TileMode.decal`.
+  Previously, `ImageFilter` defaulted to `clamp` mode if the
+  behavior wasn't specified, which sometimes surprised developers
+  as it didn't always match expectations.
+
+  As of this change, the filter automatically selects the following tile modes
+  based on context:
+
+  * `decal` with save layers and when applied to individual shape draws
+            (drawRect, drawPath, ...)
+  * `mirror` with backdrop filters.
+  * `clamp` for drawImage.
+
+## Migration guide
+
+  Only blur image filters that don't specify an explicit tile mode are
+  impacted by this change. In these cases, you can specify an explicit tile
+  mode to retain the prior behavior. However, we believe the automatic
+  selection of behavior will be a superior choice in most cases.
+
+Code before migration:
+
+```dart
+var filter = ui.ImageFilter.blur(sigmaX: 4, sigmaY: 4);
+```
+
+Code after migration:
+
+```dart
+// Set tile mode to old default.
+var filter = ui.ImageFilter.blur(sigmaX: 4, sigmaY: 4, tileMode: TileMode.clamp);
```

Comment:
Should this be turned into a lint with a fix to maintain consistent behavior?

Reply:
We want folks to use the automatic selection though and not provide a default. maybe this section isn't clear and we should just not include it?

Reply:
That intent wasn't obvious from my reading, nor was the work that has gone into creating this proposal. I'm happy to work with y'all to build a proposal that better communicates what you are attempting to achieve this, if that would be helpful. Otherwise I can suggest some minor edits?

Reply:
I mentioned this confusion in an earlier comment. I think this part of the section needs to indicate that it is optional and only needed if the developer preferred the previous behavior. I think it might be worthwhile to include it if the comment in the "after" section made it clear that nothing needs to be done, but if you really want the old behavior then this is the way to restore it. There has to be a concise way to express that the change is optional and probably not desired, but here it is if you need it.